docs/02-Install.md

```diff
@@ -391,6 +391,9 @@ The pipeline to insert BIDS datasets into LORIS currently support the
 following BIDS modalities/entities:
   - Electroencephalography ('eeg')
   - Magnetic Resonance Imaging ('anat', 'func', 'fmap', 'dwi') 
+  - Intracranial electroencephalography ('ieeg')
+      - **Note:** optional [electrical stimulation](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/04-intracranial-electroencephalography.html#electrical-stimulation) files for ieeg modality are currently not supported in LORIS-MRI (or LORIS). 
+      If a BIDS-IEEG directory includes these files, they will be ignored during the import process. 
```

Comment:
is there a reason why this is not supported? Just wondering...

Reply:
@cmadjar we didn't have any initial data to work with, and it's not in scope of the current project. 

@h-karim now that you have the basic PR in progress, can you research and outline a plan for adding electrostim - what changes/scripts are needed to support it?